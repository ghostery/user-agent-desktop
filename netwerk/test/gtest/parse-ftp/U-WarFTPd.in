ftp> open ftp.jgaa.com
Connected to ftp.jgaa.com.
220-Jgaa's official FTP server
    WarFTPd 1.81.00 (Mar  3 2002) Ready
    (C)opyright 1996 - 2002 by Jarle (jgaa) Aase - all rights reserved.
220 Please enter your user name.
Name (ftp.jgaa.com:cyp): 
230 User  logged in.
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> passive
Passive mode on.
ftp> ls
227 Entering Passive Mode (80,239,13,229,14,199)
125 Using existing ASCII mode data connection for /bin/ls (1220 bytes).
total 5127
dr-xr--r-- 1 root  root        0 Apr 22  2009 .
dr-xr--r-- 1 root  root        0 Apr 22  2009 ..
-r-xr--r-- 1 root  root   359807 Mar 13  2000 adns-0.6-win32-01.zip
-r-xr--r-- 1 root  root       77 Oct 17  2000 adns-0.6-win32-01.zip.md5
-r-xr--r-- 1 root  root   218767 Nov 11  2000 cfs-1.3.3-8bit-fix.r2.tgz
-r-xr--r-- 1 root  root     1606 Jun 13  2000 cfs-1.3.3-eight-bit-fix.diff.tar.gz
-r-xr--r-- 1 root  root     3327 Oct 19  2000 chkmailspool-1.0.tgz
-r-xr--r-- 1 root  root     9356 Oct 21  2000 chkmailspool-1.1.tgz
-r-xr--r-- 1 root  root    32742 Feb  1  2000 iisstat-1.2.zip
-r-xr--r-- 1 root  root       71 Oct 17  2000 iisstat-1.2.zip.md5
-r-xr--r-- 1 root  root    27531 Aug 14  1999 inshtml.zip
-r-xr--r-- 1 root  root       67 Oct 17  2000 inshtml.zip.md5
-r-xr--r-- 1 root  root    23780 Aug 14  1999 netcps.zip
-r-xr--r-- 1 root  root       66 Oct 17  2000 netcps.zip.md5
-r-xr--r-- 1 root  root  1057919 Nov  3  2001 openssl-0.9.6b-win32.zip
-r-xr--r-- 1 root  root    56823 Jan 28  2001 passgen-2.0.zip
-r-xr--r-- 1 root  root   826880 May 25  1997 warftpd-version2-public-source.zip
-r-xr--r-- 1 root  root       90 Oct 17  2000 warftpd-version2-public-source.zip.md5
226 Transfer complete. 1220 bytes in 0.01 sec. (119.141 Kb/s)
ftp> close
221 Goodbye. Control connection closed. 
ftp> 