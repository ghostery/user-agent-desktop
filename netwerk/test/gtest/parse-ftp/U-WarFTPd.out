<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

04-22-2009  00:00:00       <DIR>          .
04-22-2009  00:00:00       <DIR>          ..
03-13-2000  00:00:00               359807 adns-0.6-win32-01.zip
10-17-2000  00:00:00                   77 adns-0.6-win32-01.zip.md5
11-11-2000  00:00:00               218767 cfs-1.3.3-8bit-fix.r2.tgz
06-13-2000  00:00:00                 1606 cfs-1.3.3-eight-bit-fix.diff.tar.gz
10-19-2000  00:00:00                 3327 chkmailspool-1.0.tgz
10-21-2000  00:00:00                 9356 chkmailspool-1.1.tgz
02-01-2000  00:00:00                32742 iisstat-1.2.zip
10-17-2000  00:00:00                   71 iisstat-1.2.zip.md5
08-14-1999  00:00:00                27531 inshtml.zip
10-17-2000  00:00:00                   67 inshtml.zip.md5
08-14-1999  00:00:00                23780 netcps.zip
10-17-2000  00:00:00                   66 netcps.zip.md5
11-03-2001  00:00:00              1057919 openssl-0.9.6b-win32.zip
01-28-2001  00:00:00                56823 passgen-2.0.zip
05-25-1997  00:00:00               826880 warftpd-version2-public-source.zip
10-17-2000  00:00:00                   90 warftpd-version2-public-source.zip.md5
